sources/packages/backend/apps/db-migrations/src/sql/EducationProgramsOfferings/Rollback-update-offering-duplication-unique-index.sql

```diff
@@ -0,0 +1,60 @@
+DROP INDEX sims.loc_id_prog_id_offer_name_study_dts_year_study_index;
+
+-- Ensures currently duplicated offerings will be renamed to avoid issues
+-- while creating the unique constraint.
+UPDATE
```

Comment:
Do we really need an update that modifies the offering name?

I had some doubts regarding rollback impacts and had a discussion with @andrewsignori-aot

Reply:
If we have some changes to the tables and we need to rollback, yes we'll to update it to satisfy the old unique constraint. What would be your suggestion?